integration/tests/cook/test_multi_user.py

```diff
@@ -116,14 +116,14 @@ def test_job_cpu_quota(self):
         try:
             # User with no quota can't submit jobs
             with admin:
-                resp = util.set_limit(self.cook_url, 'quota', user.name, cpus=0)
+                resp = util.set_limit(self.cook_url, 'quota', user.name, cpus=0, pool=util.default_submit_pool())
                 self.assertEqual(resp.status_code, 201, resp.text)
             with user:
                 _, resp = util.submit_job(self.cook_url)
                 self.assertEqual(resp.status_code, 422, msg=resp.text)
             # User with tiny quota can't submit bigger jobs, but can submit tiny jobs
             with admin:
-                resp = util.set_limit(self.cook_url, 'quota', user.name, cpus=0.25)
+                resp = util.set_limit(self.cook_url, 'quota', user.name, cpus=0.25, pool=util.default_submit_pool())
```

Comment:
Seems like we should just make the default for all these limit-setting calls when no pool is specified be `default_submit_pool()`

Reply:
Done.